packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/InteractionLayer/InteractionLayer.js

```diff
@@ -27,105 +31,105 @@ function InteractionLayer ({
   scale,
   width
 }) {
-  const [ creating, setCreating ] = React.useState(false)
-  const { svg, getScreenCTM } = React.useContext(SVGContext)
+  const [creating, setCreating] = React.useState(false);
+  const { svg, getScreenCTM } = React.useContext(SVGContext);
 
-  function convertEvent (event) {
-    const type = event.type
+  function convertEvent(event) {
+    const type = event.type;
 
-    const svgEventOffset = getEventOffset(event)
+    const svgEventOffset = getEventOffset(event);
 
     const svgCoordinateEvent = {
       pointerId: event.pointerId,
       type,
       x: svgEventOffset.x,
       y: svgEventOffset.y
-    }
+    };
 
-    return svgCoordinateEvent
+    return svgCoordinateEvent;
   }
 
-  function getEventOffset (event) {
-    const svgPoint = svg.createSVGPoint()
-    svgPoint.x = event.clientX
-    svgPoint.y = event.clientY
-    const svgEventOffset = svgPoint.matrixTransform(getScreenCTM().inverse())
-    return svgEventOffset
+  function getEventOffset(event) {
+    const svgPoint = svg.createSVGPoint();
+    svgPoint.x = event.clientX;
+    svgPoint.y = event.clientY;
+    const svgEventOffset = svgPoint.matrixTransform(getScreenCTM().inverse());
+    return svgEventOffset;
   }
 
-  function onPointerDown (event) {
+  function onPointerDown(event) {
     if (disabled || move) {
-      return true
+      return true;
     }
 
-    const activeMark = activeTool.createMark({
-      id: cuid(),
-      frame,
-      toolIndex: activeToolIndex
-    })
-    activeMark.initialPosition(convertEvent(event))
-    setActiveMark(activeMark)
-    setCreating(true)
-    activeMark.setSubTaskVisibility(false)
-    return false
+    console.log("activeTool: ", activeTool);
+
+    if (activeTool.type) {
+      const activeMark = activeTool.createMark({
+        id: cuid(),
+        frame,
+        toolIndex: activeToolIndex
+      });
+      activeMark.initialPosition(convertEvent(event));
+      setActiveMark(activeMark);
+      setCreating(true);
+      activeMark.setSubTaskVisibility(false);
+      return false;
+    }
+    return;
```

Comment:
Recommend: if we're using an if-block, this should be `return false` to be consistent with original code